lib/improver/threshold.py

```diff
@@ -180,6 +183,38 @@ def __repr__(self):
         ).format(self.thresholds, self.fuzzy_bounds,
                  self.below_thresh_ok)
 
+    def _add_threshold_coord(self, cube, threshold):
+        """
+        Add a scalar threshold-type dimension to a cube containing thresholded
```

Comment:
coordinate not dimension?

Reply:
I did mean dimension, but the new docstring is clearer anyway.